compiler/rustc_metadata/src/rmeta/mod.rs

```diff
@@ -394,6 +394,7 @@ define_tables! {
     fn_arg_names: Table<DefIndex, LazyArray<Ident>>,
     generator_kind: Table<DefIndex, LazyValue<hir::GeneratorKind>>,
     trait_def: Table<DefIndex, LazyValue<ty::TraitDef>>,
+    trait_alias_def: Table<DefIndex, LazyValue<ty::TraitAliasDef>>,
```

Comment:
Does this amount to a breaking change to metadata that requires a bump of the `METADATA_VERSION`?